README.md

```diff
@@ -7,15 +7,15 @@
 ## Supported Specifications
 Of course, there are multiple ones to follow, which makes it trickier.
 
-* [NPM](https://npmjs.org/doc/json.html)
+* [npm](https://npmjs.org/doc/json.html)
 * [CommonJS Packages 1.0](http://wiki.commonjs.org/wiki/Packages/1.0)
 * [CommonJS Packages 1.1](http://wiki.commonjs.org/wiki/Packages/1.1)
 
 
 ## Usages
 
 ### Online
-Online copy hosted courtesy of Nick Sullivan at [http://package-json-validator.com/](package-json-validator.com). Want to run your own copy? You are welcome to clone or fork this repo.
+Online copy hosted courtesy of Nick Sullivan at <http://package-json-validator.com/>. Want to run your own copy? You are welcome to clone or fork this repo.
```

Comment:
Nit: unnecessary trailing slash. And, I think the link was meant to be the other way around from how it was before...

```suggestion
Online copy hosted courtesy of Nick Sullivan at [package-json-validator.com](https://package-json-validator.com). Want to run your own copy? You are welcome to clone or fork this repo.
```